sdk/servicebus/Microsoft.Azure.ServiceBus/src/ServiceBusConnection.cs

```diff
@@ -132,6 +132,16 @@ internal ServiceBusConnection(RetryPolicy retryPolicy = null)
         /// <remarks>Defaults to 1 minute.</remarks>
         public TimeSpan OperationTimeout { get; set; }
 
+        /// <summary>
+        /// ConnectionIdleTimeout is applied when a connection recevies no traffic for a certain period of time.
+        /// In some situations, underlying transport layer takes really long time to detect a socket error and it may
+        /// cause some operations to time out after a long wait. This timeout helps to detect and close an idle connection
+        /// without waiting for the network layer to identify a socket error. But this setting may result in some incrased
```

Comment:
@DorothySun216 it's not a nit but a good suggestion. I took it a step forward 🙂

```suggestion
        /// without waiting for the network layer to identify a socket error. But this setting may result in some increased
```